web/src/beta/lib/core/engines/Cesium/Feature/index.tsx

```diff
@@ -160,10 +160,14 @@ export default function Feature({
         {displayType.map(k => {
           const [C] = components[k] ?? [];
           const isVisible = layer.layer.visible !== false && !isHidden;
+          const useSphericalHormonicCoefficients =
+            !!props.sceneProperty.light?.sphericalHarmonicCoefficients;
```

Comment:
```suggestion
            !!props.sceneProperty.light?.sphericalHarmonicCoefficients && !!props.sceneProperty.light?.specularEnvironmentMaps;
```
I think we should handle the environment maps in here.